recipes/resiprocate/all/conanfile.py

```diff
@@ -42,11 +48,18 @@ def configure(self):
 
     def requirements(self):
         if self.options.with_ssl:
-            self.requires("openssl/1.1.1i")
+            self.requires("openssl/1.1.1j")
         if self.options.with_postgresql:
             self.requires("libpq/11.9")
         if self.options.with_mysql:
             self.requires("libmysqlclient/8.0.17")
+        if self.options.with_repro:
+            self.requires("libdb/4.8.30")
+            self.options["libdb"].with_cxx = True
```

Comment:
1. I think requires is the wrong spot
2. You need to add a check this is true in a validate method